node-operators/block-producers/operational-tasks/upgrade-your-node/upgrade-systemd.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Upgrade your Block Producer Systemd Node
+description: Follow these instructions to update your Tanssi block producer node running via Systemd to the latest version of the Tanssi client software.
+---
+
+# Upgrade Your Node Running via Systemd
+
+## Introduction {: #introduction }
+
+Keeping your node up-to-date is an important part of being a Tanssi block producer node. Not only does it help to ensure that your block producer node stays healthy, it also contributes to keeping the entire Tanssi network running smoothly. Tanssi Block Producers can subscribe to [Github Notifications](#conclusion) to be alerted for new client release versions. 
+
+This tutorial covers upgrading your Tanssi block producer node that was configured using Systemd. It assumes you have already set up your account and launched a [block producer node using Systemd](/node-operators/block-producers/onboarding/run-a-block-producer/block-producer-systemd/){target=\_blank}. 
+
+## Upgrading Your Systemd Node {: #upgrading-your-systemd-node }
+
+If you're running your block producer node via the Systemd service, you'll need to take a few steps to properly upgrade your node. In short, you'll need to stop the service, replace the Tanssi binary with the updated version, and then start the service. 
+
+You can stop your Tanssi Systemd service with the following command:
+
+```bash
+systemctl stop tanssi.service
+```
+
+Then, navigate to the directory where your Tanssi binary is stored and remove it. 
+
+```bash
+cd /var/lib/tanssi-data
+```
+
+Note, depending on the particular binary you have, your Tanssi node may be named `tanssi-node`, `tanssi-node-skylake`, or `tanssi-node-zen3`.
```

Comment:
with the `-O` flag, they should all be `tanssi-node` - but doesn't hurt to keep this